system-setup.py

```diff
@@ -49,6 +49,7 @@ def common_last(self):
         self.pip_install("-r %s/tests/pytest/requirements.txt" % ROOT)
         self.pip_install("toml")
         self.pip_install("awscli")
+        self.pip_install("gevent")
```

Comment:
It would be cleaner to list the deps inside `requirements.txt` instead of hardcoding. You could add another separate `requirements.txt` file in addition to the one from the tests.